OsmAnd/res/values/strings.xml

```diff
@@ -4196,7 +4196,7 @@
     <string name="shared_string_undo">Undo</string>
     <string name="shared_string_skip">Skip</string>
     <string name="app_name_osmand">OsmAnd</string>
-    <string name="offline_maps_and_navigation">Offline Maps\n&amp; Navigation</string>
+    <string name="offline_maps_and_navigation">Offline Maps\nand Navigation</string>
```

Comment:
We've had this forever, a changing this should be reserved to product management.

Reply:
No good tradition in the wrong use of an ampersand. If product management is not picking up on it for that long then maybe that is a good fix to find.